gpytorch/lazy/interpolated_lazy_variable.py

```diff
@@ -75,6 +75,17 @@ def __init__(
         self.right_interp_indices = right_interp_indices
         self.right_interp_values = right_interp_values
 
+    def _approx_diag(self):
```

Comment:
Two things:

1. For this to actually work in a lot of the real models, we'll need `ConstantMulLazyVariable` to also define `_approx_diag` and do the obvious thing there.

2. `ConstantMulLazyVariable` is the worst.

Reply:
(Other than point 1, looks good :-)